.circleci/config.yml

```diff
@@ -124,19 +114,6 @@ workflows:
           filters:
             branches:
               ignore: main
-      # On main
-      - run-tests-on-devices:
```

Comment:
Can this job definition be deleted now?

Reply:
I'm going to do a huge clean up of all DeviceFarm related scripts/jobs/makefile rules after I finish building the same behaviour. Currently we are missing trigger-by-commit in non-main branches